src/status_im/contexts/shell/activity_center/notification/common/view.cljs

```diff
@@ -12,11 +12,14 @@
 (defn user-avatar-tag
   [user-id]
   (let [profile (rf/sub [:contacts/contact-by-identity user-id])]
-    [quo/context-tag
-     {:blur?           true
-      :size            24
-      :full-name       (profile.utils/displayed-name profile)
-      :profile-picture (profile.utils/photo profile)}]))
+    [rn/view
+     {:on-start-should-set-responder
```

Comment:
could you pls elaboeate on how on-start-should-set-responder works here? thanks

Reply:
hi @flexsurfer, Thank you very much for reviewing PR.

We have the whole activity log as pressable for accepted requests, but when context-tag is pressed we only want that to call an on-press event and avoid propagation to parent pressable.

I picked this solution from the [stackoverflow answer](https://stackoverflow.com/questions/31866671/how-to-stop-touch-event-propagation-in-react-native), apparently adding `onStartShouldSetResponder` is enough for our situation to make it work.